setup.ps1

```diff
@@ -3,4 +3,7 @@ if (-not $env:ChocolateyInstall -or -not (Test-Path "$env:ChocolateyInstall")) {
     Invoke-Expression ((New-Object net.webclient).DownloadString("https://community.chocolatey.org/install.ps1"))
 }
 
-choco install pester -version 2.0.2

```

Comment:
The tests do need Pester 5.3.1 - should this be installed here? (And probably mention this in the README?

Reply:
Execution of the Pester tests is best done via the Vagrantfile that is provided, since it is possible that changes could be made to your system as a result of running the tests.  This is documented here: https://github.com/chocolatey/choco/blob/develop/TESTING.md#pester-tests

I notice that we don't have a link to this section from the README.md file, which we likely should have.

Reply:
I have added a new section to the README.md to provide some context and links to the TESTING.md file.